templates/docs/api.html

```diff
@@ -14,7 +14,7 @@
     </div>
   </section>
   <div class="l-docs">
-    <aside class="l-docs-sidebar" id="navigation">
+    <aside class="l-docs-sidebar" id="navigation" aria-label="document contents">
```

Comment:
In this case, this is navigation not document contents. I assume the `nav` element is sufficient here?

Reply:
We have places where there are multiple `nav` / `aside` elements on the page and in such cases we need to give user a way to differentiate between them. In my mind, "complementary - document contents" with a generic navigation inside made more sense.

Putting this aside for now, I'll need to rethink this.